client/verta/verta/operations/monitoring/alert/_entities/alert.py

```diff
@@ -182,13 +192,12 @@ def add_notification_channels(self, notification_channels):
 
         self._update(alert_msg)
 
-    # TODO: alternatively, fire() & resolve()?
-    def set_status(self, status, summary_sample=None, event_time_millis=None):
+    def set_status(self, status, event_time_millis=None):
```

Comment:
I think we should follow the example of `Summary` and `SummarySample` and use `time_utils` to accept `datetime` objects or epoch milliseconds, and by storing the `datetime` on objects and only encoding to millis for proto interactions, we can support better string representations of these objects.

Reply:
1e0094bd5c8bfccba8b584c36599efe8bd653370